tutor/templates/local/docker-compose.yml

```diff
@@ -158,7 +158,7 @@ services:
     environment:
       SERVICE_VARIANT: lms
       DJANGO_SETTINGS_MODULE: lms.envs.tutor.production
-    command: celery --app=lms.celery worker --loglevel=info --hostname=edx.lms.core.default.%%h --max-tasks-per-child=100 --exclude-queues=edx.cms.core.default
+    command: celery --app=lms.celery worker --loglevel=info --hostname=edx.lms.core.default.%%h --concurrency={{ OPENEDX_LMS_CELERY_WORKERS }} --max-tasks-per-child=100 --exclude-queues=edx.cms.core.default
```

Comment:
The docker-compose also provides mechanisms for managing replicas.

Reply:
In theory, yes.  I'd be glad to review a PR that does that instead.  I just need a way to reduce RAM usage for development. ;)